adal/src/androidTest/java/com/microsoft/aad/adal/StorageHelperTests.java

```diff
@@ -249,6 +250,9 @@ public void testVersion() throws GeneralSecurityException, IOException {
         }
     }
 
+
+    //Github issue #580. Suppress this unit test as we cannot make it work consistently.
+    @Suppress
```

Comment:
Let's keep it as suppress, this will help us if we want to add test back around the key handling.